reactive/containerd.py

```diff
@@ -232,13 +253,13 @@ def merge_custom_registries(config_directory, custom_registries,
     :return: List Dictionary merged registries
     """
     registries = []
-    registries += json.loads(custom_registries)
+    registries += _registries_list(custom_registries, default=[])
```

Comment:
What's the `default=[]` here for? Presumably if we made it this far, then `invalid_custom_registries` has been called and we know the config is valid, right? And if for some reason it's not, shouldn't we halt?

Reply:
I found situations where the `old_custom_registries` weren't valid JSON or weren't lists.  This could be true on an upgrade to this charm. 
I am using this for situations where those were causing permanent failures from the config_change hook completing.